allow showing any content-type in calendar (#11)
mbarde/volto-fullcalendar-block: README.md

```
More details checkout the following link: https://5.docs.plone.org/external/plone.app.dexterity/docs/advanced/catalog-indexing-strategies.html#adding-new-indexes-and-metadata-columns

> ### Add content-types other than Events in the Calendar

Reviewer: ```suggestion
### Add content types other than events in the calendar
```

> You may have developed a custom version of an Event that want to show in a calendar, or you may have other content-types that want to show in a calendar.

Reviewer: ```suggestion
You may have developed a custom version of an event or other content types that want to show in a calendar.
```

> To do so, you need to provide a function that returns the date, title and urls to show for your content-type.

Reviewer: ```suggestion
To do so, you need to provide a function that returns the date, title, and URLs to show for your content type.
```

> You need to register that function in the block config in your addon this way:

Reviewer: ```suggestion
You need to register that function in the block configuration in your add-on this way:
```

```js
config.blocks.blocksConfig.fullcalendar.contentConverters['YourContentType'] = (item) => {
  return {
    title: item.title,
    start: item.XXX, # <- point this attribute to the one that provides the start date
    end: item.XXXX,  # <- point this attribute to the one that provides the end date
    url: flattenToAppURL[item['@id']] # <- point this attribute to the one that provides the url you want to go when clicking the event in the calendar
  }
};

```

> The block already includes a default function for the Event content-type.

Reviewer: ```suggestion
The block already includes a default function for the event content type.
```
